collector/receiver/telemetryapireceiver/receiver.go

```diff
@@ -71,6 +78,8 @@ type telemetryAPIReceiver struct {
 	types                   []telemetryapi.EventType
 	resource                pcommon.Resource
 	faasFunctionVersion     string
+	faasName                string
+	faaSMetricBuilders      *FaaSMetricBuilders
```

Comment:
debatble nit - naming
```suggestion
	faasMetricBuilders      *FaasMetricBuilders
```
lmk what you think.

Reply:
This is how the semconv library has it which is why I've used FaaS since its short for "Function as a Service"